app/src/main/res/values/strings.xml

```diff
@@ -168,10 +168,10 @@
     <string name="request_timeout">Request Timeout</string>
     <string name="superuser_notification">Superuser Notification</string>
     <string name="settings_su_reauth_title">Reauthenticate after upgrade</string>
-    <string name="settings_su_reauth_summary">Reauthenticate superuser permissions after an application upgrades</string>
```

Comment:
Not consistent with "app/s" elsewhere.
No apps upgrade on their own, but this is suggested.

Reply:
I agree the use of "an" is Incorrect here with the plural.

Reply:
Do you also agree that upgrading apps is not an emergent phenomenon?

Reply:
I mean sometimes they do have an auto-updater built in, so technically not incorrect. 😉

Reply:
Good point.
Also trying to imply the vastly more common scenario.

Reply:
Actually I changed my mind, removing the "an" here makes it unclear whether it refers to the Magisk app or apps in general. To correctly understand the latter the "an" must be put back.